py/selenium/webdriver/remote/webdriver.py

```diff
@@ -171,6 +172,8 @@ def __init__(
         keep_alive: bool = True,
         file_detector: Optional[FileDetector] = None,
         options: Optional[Union[BaseOptions, List[BaseOptions]]] = None,
+        locator_converter: Optional[LocatorConverter] = None,
+        web_element_cls: Optional[type] = None,
```

Comment:
What does `type` mean here?

Reply:
It means that `web_element_cls` can be of any class type, if we want to be precise we can have this:
`web_element_cls: Optional[Type[WebElement]] = None`

What do you suggest?

Reply:
I am not a python expert so I was unsure what this means. I guess it's fine this way.